docs/source/basic/data_prep.md

```diff
@@ -333,23 +407,6 @@ def part_filter(part: str) -> bool:
 
 For more information please also read [](custom-sample-loader).
 
-(wds-format-special)=
-### Special format
-Sometimes, your data will not be easily represented as a `field_map` explained above. 
-For example, your data may contain
-
-* structured data like nested boxes for each sample
-* custom binary formats
-* xml / html / pickle etc.
-
-In those cases you have two options:
-
-1. Creating a custom `sample_loader.py` in the `.nv-meta` folder as explained above.
-    * This will typically do the job and is preferred if you only have to do some small conversions.
-2. Using a `CrudeWebdataset`
-    * For more intricate conversions, you can use a CrudeWebdataset that will pass your samples in a raw form into your TaskEncoder where you can then convert them based on the subflavor for example. For more details see [](crude-data).
-
-Even for these specific wds formats, you would start preparing your data using the [dataset preparation command](#energon_data_prepare), but you will need to define a custom sample loader or select `CrudeWebdataset` in the dataprep wizard. 
 
 (data-on-disk)=
```

Comment:
Maybe we should move this to it's own advanced section? Too many details for "basic"